CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml

```diff
@@ -2441,6 +2448,27 @@
                                                    TextTrimming="CharacterEllipsis" />
                                     </Grid>
                                 </Button>
+                                <Button x:Name="SyncDbPlaytimeBtn"
+                                        Margin="-4,0"
```

Comment:
Margin of `SyncDbPlaytimeBtn` should be `-4,0,-4,-4`, and `ResetPlaytimeButton` margin `-4,0`

Reply:
Fixed